### ISSUE TITLE
[Request] CLI

### ISSUE THREAD

**Issue by rayaman** [NguyenDanPhuong/MangaRipper#80]
do you think a command line interface would be possible where one could do:
mangaripper.exe url_to_manga outputdir

**Comment by 0xh3LL0** [comment]
idk if this will solve you problem, but if you prefer CLI you can use [manga_downloader](https://github.com/jiaweihli/manga_downloader) instaed

**Comment by NguyenDanPhuong** [comment]
Yes, it's possible, the component already separated. It needs to create a new CLI project and parse parameter then call the existing component. But I quite lack of time, if any one interested in this. Please go ahead and create pull request.
